### ISSUE TITLE
Migration to firefox-devtools org

### ISSUE THREAD

**Issue by digitarald** [hbenl/vscode-firefox-debug#108]
Firefox DevTools wants to support the development of this extension.

The people involved, apart from @hbenl and me, are @janodvarko and @jasonLaster.

**How will shared governance help this project?**

1. Keep the extension in sync with Firefox's debugger server, remote protocol changes and upcoming features (e.g. dfddc7a8134b3672fd9857f4c1d39a56a582ef02, 5f86bdbd2c27137e55f3bfd3352a95abb7a2cb6a, etc)
2. Simplify setup and integration with DevTools
3. Make this the official Firefox extension for VSCode, with proper branding (e.g. #81)
5. Do more joint outreach to spread the word about the extension and how to set up

**What's next?**

🚧 _Draft_

- [ ] Add Mozilla's `contributing.md`, `CODE_OF_CONDUCT.md`
- [ ] Transfer repo to https://github.com/firefox-devtools
  - [ ] Make @hbenl and @jasonLaster, @janodvarko & @digitarald maintainers of that repo
  - No need to update Copyright/License
- [ ] Update references
  - [ ] VSCode extension marketplace
- [ ] Update branding
  - [ ] New icon
  - [ ] Updated description
- [ ] Ship a new release
- [ ] Announcement
  - [ ] Blog post

**Comment by hbenl** [comment]
I have one more item for the TODO list: every maintainer should be able to publish new versions of the extension. There seem to be 2 options for this:
* move the extension to a new publisher account and add all maintainers to this publisher
* add other maintainers to the current publisher account

The big downside of the first approach is that this would change the extension ID in the VS Code Marketplace and hence all current users of the extension would have to reinstall it - it seems there is currently no way to automatically migrate them to the new extension ID.
The downside of the second approach is that the current publisher account is also used for my other extensions and I would give you the ability to manage those extensions as well. But I'd trust you to not abuse this, so I'd be OK with that.

There doesn't seem to be any documentation on how to do this, all I could find so far are these issues:
[Transfer Publishers in Marketplace](https://github.com/Microsoft/vscode/issues/3670)
[Guidance for transferring ownership of extension](https://github.com/Microsoft/vscode/issues/21478)

**Comment by digitarald** [comment]
Good point @hbenl ! I have been in touch with the VSCode team and they offered to change the publisher ID. I will add those items to the list then.

**Comment by hbenl** [comment]
The migration of the extension to the firefox-devtools publisher was successful and I already made a bugfix release with the new publisher.
